ui-src/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "automated-perf-test",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "build": "ng build --prod",
+    "watch": "ng build --watch",
+    "test": "ng test",
+    "lint": "ng lint",
```

Comment:
@angerooney11  & @ColmCooney  `ng lint` Can you run this and fix the issues that arise, it should go a long way to fixing most of the issues.